ui/src/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentEditor.js

```diff
@@ -10,6 +11,31 @@ export class DocumentEditor extends Component {
     error: {},
   };
 
+  get documentRequest() {
+    const request = _get(this.props, 'location.state', null);
+    if (request) {
+      const data = {
+        documentRequestPid: request.pid,
+        metadata: {
+          // NOTE: Cheating so the authors don't break our document form
+          authors: [{ full_name: request.metadata.authors }],
+          title: request.metadata.title,
+        },
+      };
+      const journalTitle = _get(request, 'metadata.journal_title', null);
+      if (journalTitle) data.metadata.journal_title = journalTitle;
+
+      const edition = _get(request, 'metadata.edition', null);
+      if (edition) data.metadata.edition = edition;
```

Comment:
Great catch, apparently we did that because the forms throw errors when we supply `null` values. If we provide `undefined` though they work as expected.